kafka/conn.py

```diff
@@ -778,7 +778,8 @@ def filter(self, record):
         log.addFilter(log_filter)
 
         test_cases = [
-            ((0, 10), ApiVersionRequest[0]()),
+            ((0, 10, 1), MetadataRequest[2]([])),
```

Comment:
test via `KAFKA_VERSION=0.8.0 tox -e py27 -- test/test_consumer_group.py::test_paused` -- this hangs indefinitely on my laptop